Detections/MultipleDataSources/AdditionalFilesUploadedByActor.yaml

```diff
@@ -0,0 +1,95 @@
+id: b9e3b9f8-a406-4151-9891-e5ff1ddd8c1d
+name: Linked Malicious Storage Artifacts
+description: |
+  'An IP address which uploaded malicious content to an Azure Blob or File Storage container (triggering a malware alert) also uploaded additional files.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: MicrosoftCloudAppSecurity
+    dataTypes:
+      - SecurityAlert
+queryFrequency: 15m
```

Comment:
Are we sure we want to run this every 15 minutes?  Also, seems like you would not want to run every 15 mins and look back at 30 mins, otherwise I think this would cause duplicates.  Does it make sense to move this to daily and then bin on 30 mins?

Reply:
We do not need to do that at all, this was a remnant of the test environment that I didn't change. I think we can run daily, and I don't think we need to bin, as it's triggered based on an alert, any upload activity from the actor IP in a day seems OK to me?